cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java

```diff
@@ -505,161 +536,243 @@ private void _convertToThemeProject(File themePlugin) {
 		}
 	}
 
-	private void _convertToWarProject(File warsDir, File pluginDir, boolean removeSource) {
-		try {
-			warsDir.mkdirs();
+	private void _convertToWarProject(File pluginsSdkDir, File warsDir, File pluginDir, boolean removeSource)
+		throws Exception {
 
```

Comment:
This is a rather large method, probably not a big deal right now but eventually I think we should consider refactoring this.